sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -145,9 +252,15 @@ def plot(
             Axes object to plot on. If `None`, a new figure and axes is
             created.
 
-        name : str, default=None
-            Name of precision recall curve for labeling. If `None`, use
-            `estimator_name` if not `None`, otherwise no labeling is shown.
+        names : str, default=None
+            Names of each precision-recall curve for labeling. If `None`, use
+            name provided at `PrecisionRecallDisplay` initialization. If not
+            provided at initialization, no labeling is shown.
```

Comment:
It seems reasonable that if we change the `name` parameter in the class init, we should change it here to, especially as we don't advocate people to use `plot` directly.